src/main/java/com/github/fge/jsonpatch/DualPathOperation.java

```diff
@@ -48,8 +57,9 @@
     protected DualPathOperation(final String op, final JsonPointer from,
         final JsonPointer path)
     {
-        super(op, path);
+        this.op = op;
```

Comment:
Let's keep it for now and remove it in a different change.  It is useful for serialization of all "dual-path" operations, but we can consider serializing in a different way (@JsonProperty annotations?).